### PR TITLE
feat(admin): allow to import all SPDX licenses

### PR DESCRIPTION
**Pull request by maximilianhuber**
This adds a button in the admin db ui which allows an admin to import all SPDX licenses into the DB

### CODE REVIEW

**Comment by alex-evo** [comment]
@maximilianhuber there's a build error. Probably due to a faulty rebase.

**Comment by maximilianhuber** [comment]
I will have a look.

File: backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+     * @return
+     */
+    public static boolean matchesSpdxLicenseText(License license) {
+        return matchesSpdxLicenseText(license, license.getShortname());
+    }
+
+    public static boolean matchesSpdxLicenseText(License license, String spdxId) {
+        final Optional<SpdxListedLicense> spdxLicense = getSpdxLicense(spdxId);
+        if(!spdxLicense.isPresent()) {
+            return true;
```
Comment (alex-evo) [inline backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java]: Does it mean that any `license` matches its SPDX counterpart when that SPDX counterpart is not found? Looks like a mistake to me.

File: backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        return  LicenseCompareHelper.isLicenseTextEquivalent(license.getText(), licenseText);
+    }
+
+    /**
+     * returns a list of potentially matching SPDX license IDs
+     * @param license
+     * @return
+     * @throws SW360Exception
+     */
+    public static List<String> findMatchingSpdxLicenseIDs(License license) throws SW360Exception {
```
Comment (alex-evo) [inline backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java]: These last four methods in the class appear to be never used. What's the point?

Comment (maximilianhuber) [inline backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java]: These functions were implemented for a proof-of-concept and might be used in another feature.

File: backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java (hunk 1 above)
Comment (maximilianhuber) [inline backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java]: A license, which has no SPDX counterpart, does **not not** match the SPDX-license.

This function should prevent SPDX-incompatilities, and not only allow SPDX licenses

Comment (alex-evo) [inline backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/tools/SpdxConnector.java]: Okay, I got it. Could you, perhaps, add some code comment to clarify this point inline? Just to accommodate my short memory span.

**Pull request merged**